erpnext_com/www/docs/user/manual/en/loan-management/loan-security-pledge.md

```diff
@@ -0,0 +1,44 @@
+<!-- add-breadcrumbs -->
+# Loan Security Pledge
+> Introduced in Version 13
+
+**Loan Security Pledge contains the loan security and their qty pledged against a loan.**
+
+<img class="screenshot" alt="Make Loan Security Pledge" src="{{docs_base_url}}/assets/img/loan-management/loan-security-pledge-flow.png">
+
+To access the Loan Security Pledge list, go to:
+> Home > Loan Management > Loan > Loan Security Pledge
+
+
+<img class="screenshot" alt="Loan Application" src="{{docs_base_url}}/assets/img/loan-management/loan-security-pledge.png">
+
+## 1. Prerequisites
+Before creating and using a Loan Security, it is advised that you create the following first:
+
+* [Loan Security Type](/docs/user/manual/en/loan-management/loan-security-type)
+* [Loan Security](/docs/user/manual/en/loan-management/loan-security)
+* [Loan Application](/docs/user/manual/en/loan-management/loan-application)
+
+## 2. How to Create a Loan Security Pledge
+1. Go to Loan Security Pledge List, click on New.
+2. Select applicant type.
+3. Select the applicant.
+4. Select Company
+4. Enter the Loan Securities and their quantities to be delivered in the securities table. The Loan Security Price will be fetched from the item master and the amount will be auto calculated.
+6. Click "Save" to save the draft of the Loan Security Pledge.
+7. Click "Submit" to submit the Loan Security Pledge.
+8. Once the Loan Security Pledge is Submitted it is ready to be Pledged Against a Loan
+
+### 2.1. Other ways to create a Loan Security Pledge
+1. You can also create a Loan Security Pledge from a approved Loan Application via the Create button on the top right
+
+<img class="screenshot" alt="Loan Application" src="{{docs_base_url}}/assets/img/loan-management/create-loan-security-pledge.png">
+
+2. Or you can also create a Loan Security Pledge from the [Loan Security Shortfall](/docs/user/manual/en/loan-management/loan-security-shortfall) in case of a loan security to value shortfall by clicking on the Add Loan Security button on the top right
```

Comment:
```suggestion
2. Or you can also create a Loan Security Pledge from the [Loan Security Shortfall](/docs/user/manual/en/loan-management/loan-security-shortfall) in case of a loan security to value shortfall by clicking on the **Add Loan Security** button on the top right
```